Essentials/src/messages.properties

```diff
@@ -296,7 +296,8 @@ msgDisabled=\u00a76Receiving messages \u00a7cdisabled\u00a76.
 msgDisabledFor=\u00a76Receiving messages \u00a7cdisabled \u00a76for \u00a7c{0}\u00a76.
 msgEnabled=\u00a76Receiving messages \u00a7cenabled\u00a76.
 msgEnabledFor=\u00a76Receiving messages \u00a7cenabled \u00a76for \u00a7c{0}\u00a76.
-msgFormat=\u00a76[\u00a7c{0}\u00a76 -> \u00a7c{1}\u00a76] \u00a7r{2}
```

Comment:
This will suddenly break people's message formatting without warning - is this what we want?